tests/hpc/pdsh_slave.pm

```diff
@@ -13,7 +13,7 @@
 # Maintainer: soulofdestiny <mgriessmeier@suse.com>
 # Tags: https://fate.suse.com/321714
 
-use base "hpcbase";
+use base "opensusebasetest";
```

Comment:
There's a call to `$self->enable_and_start` in line 40.

Reply:
Changed!